### PR TITLE
Update Cadence to v0.35.0

### PR DESCRIPTION
**Pull request by turbolent**
## Description

Automatically update to:
- [onflow/cadence v0.35.0](https://github.com/onflow/cadence/releases/tag/v0.35.0)

### CODE REVIEW

**Review by dsainati1 — APPROVED** [review]

**Review by SupunS — APPROVED** [review]

File: access/grpc/convert_test.go [hunk 1]
```diff
@@ -202,6 +202,8 @@ func TestConvert_Event(t *testing.T) {
 	eventB, err := messageToEvent(msg, nil)
 	require.NoError(t, err)
 
+	_ = eventB.Value.Type().ID()
```
Comment (sideninja) [inline access/grpc/convert_test.go:205]: what does this do?

**Review by sideninja — APPROVED** [review]
Thank you, left a question.

File: access/grpc/convert_test.go (hunk 1 above)
Comment (turbolent) [reply access/grpc/convert_test.go:205]: Good question, I'll add comments

Comment (turbolent) [reply access/grpc/convert_test.go:207]: Added in 21631e000c73baca6c87422d031a7c4e38f6275d

**Pull request merged**